### PR TITLE
Resolve triple dot false negative in rule 930100

### PR DESCRIPTION
**Pull request by franbuehler**
This PR resolves issue #2205, a false negative with triple dot (`/.../`), by extending the current regex with a `{2,3}` instead of only `{2}`. 
This PR also adds a positive testcase.

### CODE REVIEW

**Review by RedXanadu — COMMENTED** [review]
This worked as intended for me on both the modsec2-apache and modsec3-nginx containers.

I wonder if the `msg` should be updated? It currently prints `Path Traversal Attack (/../)` to the error log, which might be misleading in response to a _triple_ dot attack. What do you think @franbuehler?

File: tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml [hunk 1]
```diff
@@ -21,3 +21,18 @@
...
+      stages:
+        -
+          stage:
+            input:
+              dest_addr: "127.0.0.1"
+              method: "GET"
+              port: 80
+              headers:
+                  Host: "localhost"
+              uri: "/?foo=.../.../WINDOWS/win.ini"
```
Comment (RedXanadu) [inline tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml:36]: Should the test for this rule use encoded dots and forward slashes to match the rule's regular expression?

Comment (franbuehler) [reply tests/regression/tests/REQUEST-930-APPLICATION-ATTACK-LFI/930100.yaml:36]: I also added a test with encoded slashes and dots.

**Comment by franbuehler** [comment]
Thank you for this welcome suggestion, @RedXanadu. I updated the rules message and added a new test with encoded slashes and dots. What do you think about it?

**Comment by RedXanadu** [comment]
Looks good! Re-tested the test with the new encoded pattern, works as expected.

Merging now.

**Pull request merged**